apps/filebrowser/src/filebrowser/templates/fb_components.mako

```diff
@@ -43,7 +43,7 @@ else:
         %elif path.lower().find('gs://') == 0:
           <li style="padding-top: 12px">
             <span class="breadcrumb-link homeLink">
-              <svg class="hi"><use xlink:href='#hi-adls'></use></svg>
+              <svg class="hi"><use xlink:href='#hi-gs'></use></svg>
```

Comment:
xlink:href is deprected, but I guess the usage is all over the place in hue. See https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href

Reply:
Nice catch. Will fix separately.